git_machete/client.py

```diff
@@ -1847,68 +1859,92 @@ def check_that_forkpoint_is_ancestor_or_equal_to_tip_of_branch(
+
         debug('checkout_github_pr()', f'organization is {org}, repository is {repo}')
-        print(f"Fetching {remote}...")
+        if verbose:
+            print(f"Fetching {remote}...")
         self.__git.fetch_remote(remote)
 
-        pr = get_pull_request_by_number_or_none(pr_no, org, repo)
-        if not pr:
-            raise MacheteException(f"PR #{pr_no} is not found in repository `{org}/{repo}`")
-        if pr.full_repository_name:
-            if '/'.join([remote, pr.head]) not in self.__git.get_remote_branches():
-                remote_already_added: Optional[str] = self.__get_added_remote_name_or_none(pr.repository_url)
-                if not remote_already_added:
-                    remote_from_pr: str = pr.full_repository_name.split('/')[0]
-                    if remote_from_pr not in self.__git.get_remotes():
-                        self.__git.add_remote(remote_from_pr, pr.repository_url)
-                    remote_to_fetch: str = remote_from_pr
-                else:
-                    remote_to_fetch = remote_already_added
-                if remote != remote_to_fetch:
-                    print(f"Fetching {remote_to_fetch}...")
-                    self.__git.fetch_remote(remote_to_fetch)
-                if '/'.join([remote_to_fetch, pr.head]) not in self.__git.get_remote_branches():
-                    raise MacheteException(f"Could not check out PR #{pr_no} because its head branch `{pr.head}` is already deleted from `{remote_to_fetch}`.")
-        else:
-            warn(f'Pull request #{pr_no} comes from fork and its repository is already deleted. No remote tracking data will be set up for `{pr.head}` branch.')
-            print(fmt(f"Checking out `{pr.head}` locally..."))
-            checkout_pr_refs(self.__git, remote, pr_no, LocalBranchShortName.of(pr.head))
-            self.flush_caches()
-        if pr.state == 'closed':
-            warn(f'Pull request #{pr_no} is already closed.')
-        debug('checkout_github_pr()', f'found {pr}')
-
-        all_open_prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
-
-        path: List[LocalBranchShortName] = self.__get_path_from_pr_chain(pr, all_open_prs)
-        reversed_path: List[LocalBranchShortName] = path[::-1]  # need to add from root downwards
-        for index, branch in enumerate(reversed_path):
-            if branch not in self.managed_branches:
-                if index == 0:
-                    self.add(
-                        branch=branch,
-                        opt_as_root=True,
-                        opt_onto=None,
-                        opt_yes=True)
-                else:
-                    self.add(
-                        branch=branch,
-                        opt_onto=reversed_path[index - 1],
-                        opt_as_root=False,
-                        opt_yes=True)
+        pr: Optional[GitHubPullRequest] = None
+        for pr in prs_numbers:
```

Comment:
One thing! As for now, AFAICS the PRs are added in reversed order for some reason:

```
$ git machete github checkout-prs --by amalota
remote: Enumerating objects: 183, done.
remote: Counting objects: 100% (183/183), done.
remote: Compressing objects: 100% (35/35), done.
remote: Total 175 (delta 141), reused 173 (delta 139), pack-reused 0
Receiving objects: 100% (175/175), 13.75 KiB | 160.00 KiB/s, done.
Resolving deltas: 100% (141/141), completed with 7 local objects.
From github.com:VirtusLab/git-machete
 + e6475c6...c26f33b ci/update_contributing_new_uid_gui -> origin/ci/update_contributing_new_uid_gui  (forced update)
Pull request #416 checked out at local branch cli/checkout_prs_zsh_bash_completion
Pull request #410 checked out at local branch ci/update_contributing_new_uid_gui_5
Pull request #409 checked out at local branch ci/update_contributing_new_uid_gui
Pull request #408 checked out at local branch update_contributing
Pull request #407 checked out at local branch circle_ci_forbidden_access
```

I'd rather use increasing order... the resulting trees are ofc isomorphic (since parent-child relation are still reflected correctly), but usually having **sibling** branches ordered by **increasing** PR number makes more sense